Makefile

```diff
@@ -38,6 +38,17 @@ IMAGE_TAG_BASE ?= k8ssandra/k8ssandra-operator
 # You can use it as an arg. (E.g make bundle-build BUNDLE_IMG=<some-registry>/<project-name-bundle>:<tag>)
 BUNDLE_IMG ?= $(IMAGE_TAG_BASE)-bundle:v$(VERSION)
 
+# BUNDLE_GEN_FLAGS are the flags passed to the operator-sdk generate bundle command
+BUNDLE_GEN_FLAGS ?= -q --overwrite --version $(VERSION) $(BUNDLE_METADATA_OPTS)
```

Comment:
This code isn't used anywhere.

Reply:
Right, I need to reference the new variable when calling `generate bundle` in the `bundle` target.
Btw this also need to be done in cass-operator.

Reply:
Yep, though it's not high severity (k8ssandra-operator does not use bundles and cass-operator uses separate script when digests are used)